app/scripts/views/AppView.js

```diff
@@ -35,6 +35,7 @@ define([  'backbone',
 
       this.model.get('workspaces').on('add', this.addWorkspaceTab, this);
       this.model.get('workspaces').on('remove', this.removeWorkspaceTab, this);
+      this.model.get('workspaces').on('hide', this.hideWorkspaceTab, this);
       this.model.on('change:showingSettings', this.viewSettings, this);
       this.model.on('change:showingFeedback', this.viewFeedback, this);
       this.model.on('change:showingHelp', this.viewHelp, this);
```

Comment:
We should stop this practice.  `Model.on` is bad practice.  Do you mind fixing these?

http://stackoverflow.com/questions/16823746/backbone-js-listento-vs-on